app/views/pkg/versions/version_row.mustache

```diff
@@ -6,13 +6,17 @@
   <td><strong><a href="/packages/{{package}}/versions/{{version}}">{{version}}</a></strong></td>
   <td>{{short_created}}</td>
   <td class="documentation">
-    <a href="{{& documentation_url}}"
+    <a class="version-table-icon" href="{{& documentation_url}}"
        title="Go to the documentation of {{package}} {{version}}">
       <img src="{{& icons.documentation}}" alt="Go to the documentation of {{package}} {{version}}" />
     </a>
+    <span class="version-table-status">
+      {{#dartdoc_ok}}(OK){{/dartdoc_ok}}
```

Comment:
I'm not sure we want to explicitly show an OK status - that should be the expectation. I'd prefer just showing the failed status, if applicable.